### ISSUE TITLE
R/E tab - "Show Additional Data Tables" checkbox not accessible

### ISSUE THREAD

**Issue by humoroussmile** [CSGJusticeCenter/MCLCShiny#68]
@MarEichler the checkbox for showing additional data tables does not appear to be accessible. While I can keyboard tab to it, I cannot toggle it with the ENTER key on the keyboard. Can you look into this? I'm using the app on firefox. Thanks!
